tests/acceptance/01_vars/01_basic/fqhost_domain.cf

```diff
@@ -0,0 +1,33 @@
+##############################################################
+#
+# Test default:sys.fqhost with domain CFE-4053
+#
+##############################################################
+
+body common control
+{
+  domain => "cfengine.com";
+  bundlesequence => { "check" };
+}
+
+bundle agent __main__
+{
+  methods:
+    "check";
+}
+
+##############################################################
+
+bundle agent check
+{
+  reports:
+    DEBUG::
+      "$(sys.fqhost), $(sys.uqhost).cfengine.com";
+
+    any::
+      "$(this.promise_filename) Pass"
+        if => strcmp( "$(sys.fqhost)", "$(sys.uqhost).cfengine.com" );
+
+      "$(this.promise_filename) FAIL"
+        unless => strcmp( "$(sys.fqhost)", "$(sys.uqhost).cfengine.com" );
```

Comment:
I don't expect this to work on my machine. And many other machines...

Reply:
Why is this?

Reply:
Because I don't think my machine has `$(sys.uqhost).cfengine.com` as its FQDN. Or are we ensuring that somewhere?

Reply:
Oh! I guess we are! I didn't see that. Or rather, I didn't even know it was possible. :sweat_smile: